scss/_variables.scss

```diff
@@ -648,10 +648,30 @@ $focus-visible-outer-offset:          $focus-visible-inner-width !default; // OU
 $focus-visible-outer-color:           $black !default; // OUDS mod
 // scss-docs-end focus-visible-variables
 
+// fusv-disable
+// scss-docs-start box-shadow-variables-ouds
```

Comment:
wondering if this part should be inside the semantic file or variables file.

Reply:
Maybe `box-shadow-variables-ouds` should rather be `elevation-variables-ouds`